backend/src/main/java/gov/cdc/usds/simplereport/config/AuthorizationConfiguration.java

```diff
@@ -95,11 +95,11 @@
   private static final String SPEL_CAN_VIEW_TEST_EVENT =
       "@" + AUTHORIZER_BEAN + ".userCanViewTestEvent(#testEventId)";
 
-  private static final String SPEL_CAN_VIEW_TEST_ORDER =
-      "@" + AUTHORIZER_BEAN + ".userCanViewTestOrder(#testOrderId)";
+  private static final String SPEL_CAN_VIEW_QUEUE_ITEM =
+      "@" + AUTHORIZER_BEAN + ".userCanViewQueueItem(#testOrderId)";
```

Comment:
Are test orders and queue items distinct entities, or is this just a clarifying change?

Reply:
queue items are test orders that are still pending e.g. they haven't been submitted yet. it's a substantive difference.